x-pack/plugins/kubernetes_security/public/components/alerts_list/index.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButtonIcon } from '@elastic/eui';
+import { CSSObject } from '@emotion/react';
+
+const alertsListContainer: CSSObject = {
+  position: 'relative',
+  height: '180px',
+  padding: '16px',
```

Comment:
I'm assuming the conversion from fixed units to [euiTokens](https://elastic.github.io/eui/#/theming/sizing) will be done in future iterations?